Edits to team, adtos and tos md files

sindhus opened the pull request:
Added current team, link to their twitter
Edited sentence phrasing in adtos and tos

Commit:
WooO!

Commit:
Edits to team, adtos and tos md files

Added current team, link to their twitter
Edited sentence phrasing in adtos and tos

harisibrahimkv commented on hasweb/pages/about/team.md:
```diff
@@ -1 +1,12 @@
+Kiran Jonnalagadda [@jackerhack](https://twitter.com/jackerhack)
+Zainab Bawa [@zainabbawa](https://twitter.com/zainabbawa)
+Vidya Ramakrishnan [@vidya_ramki](https://twitter.com/vidya_ramki)
+Shreyas Satish [@shreyas_satish](https://twitter.com/vidya_ramki)
```
Edit Shreyas' Twitter handle.

jace commented on hasweb/pages/about/team.md:
```diff
@@ -1 +1,12 @@
+Shreyas Satish [@shreyas_satish](https://twitter.com/shreyas_satish)
+Karthik Balakrishnan [@karthikb351](https://twitter.com/karthikb351)
+Jyothsna Madabusi 
+Hari C M (@haricm](http://twitter.com/haricm)
```
Incorrect bracket type

jace commented on hasweb/pages/about/team.md:
```diff
@@ -1 +1,12 @@
+Karthik Balakrishnan [@karthikb351](https://twitter.com/karthikb351)
+Jyothsna Madabusi 
+Hari C M (@haricm](http://twitter.com/haricm)
+Sindhu Sundar [@sindhu](http://twitter.com/sindhu)
```
Please sort names alphabetically.

jace commented on hasweb/pages/site/tos.md:
```diff
@@ -46,7 +50,7 @@ The terms of service specific to posting a listing are listed under the [Job Boa
 
 * Events
 
-HasGeek organizes events e.g. conferences, workshops, hacknights for the technology community, and incurs expenses for venue, food, snacks, stationery & printing, audio & video equipment as well as other rental or purchased equipment. HasGeek reserves entry to select events only to valid ticket holders. A ticket is valid only for the persons named on the ticket issued by us.
```
Validity clause shouldn't be removed. Tickets are not transferable without authorization. Among other things, this is a legal requirement for liability in case of an incident -- we need to know who we are hosting.

sindhus replied:
I haven't removed the validity clause entirely. I have only removed the line "A ticket is valid only for the persons named on the ticket issued by us." which isn't true because we issue tickets in name of Company as well. 

Since the whole clause is a single line, Git thinks I've removed the entire line instead of a part of the line.

jace commented on hasweb/pages/site/tos.md:
```diff
@@ -79,11 +83,12 @@ If you violate these terms, HasGeek may discontinue providing you one or more or
 In short, WE CAN EXCOMMUNICATE YOU. HAUL YOUR ASS TO COURT. TAKE YOU TO THE COPS.
 
+TL;DR: We *can* sue you.
+
```
Please remove this and the previous line. It's hostile language.

sindhus replied:
"haul your ass" is not hostile? :)

jace replied:
That's why I said remove it. :)

Commit:
Update hasjob tos link

Commit:
Updated job board tos

Commit:
Update privacy.md

Add terms specific to Hasjob
Add section about 3rd party services we use setting cookies

Commit:
Update index.md

Commit:
Move privacy.md to hjprivacy.md

because we now have per-website privacy policy

Commit:
Update contact.md

Commit:
Update index.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/hasgeek/hasweb?pullRequest=21) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/hasgeek/hasweb?pullRequest=21) before we can accept your contribution.<br/><hr/>**Sindhu S** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/hasgeek/hasweb?pullRequest=21) it.</sub>
